cmd/add/hooks/hooks.go

```diff
@@ -0,0 +1,108 @@
+package hooks
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/loginradius/lr-cli/api"
+	"github.com/spf13/cobra"
+)
+
+var Name string
+var Event string
+var eventOption string
+var TargetUrl string
+
+func NewHooksCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "hooks",
+		Short: "Adds hooks",
+		Long: heredoc.Doc(`
+		This command adds webhooks which are configured to an App.
+		`),
+		Example: heredoc.Doc(`
+			$ lr adds hooks
+
+			Webhook has been added.
+ 
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return addHooks()
+		},
+	}
+	return cmd
+}
+
+func addHooks() error {
+	checkInput := input()
+	if !checkInput {
+		fmt.Println("Please enter the input paramaters properly.")
+		return nil
+	}
+	checkPlan, err := api.CurrentPlan()
+	if err != nil {
+		return err
+	}
+	if !checkPlan {
+		return nil
+	}
+	err = add()
+	if err != nil {
+		return err
+	}
+	fmt.Println("Webhook has been added.")
+
+	return nil
+}
+
+func input() bool {
+	fmt.Printf("Enter Name: ")
+	fmt.Scanf("%s", &Name)
+	if Name == "" {
+		fmt.Println("Name is a required entry")
+		return false
+	}
+	event := map[string]string{
+		"1": "Login",
+		"2": "Register",
+		"3": "ResetPassword",
+		"4": "UpdateProfile",
+	}
```

Comment:
I can see there are 6 events in the dashboard, why we have added only 4?

Reply:
Developer plan has only 4 options whereas the Developer Pro plan has 6 options for events. Currently, only the developer plan options have been included. This can be extended to Developer Pro plan at a later time.